### PR TITLE
Add an additional method of dumping and some fixes to the setup guide

### PR DESCRIPTION
**Pull request by DerekTurtleRoe**
I added instructions for dumping with the frontend for DiscImageCreator (MPF) as well as a screenshot to give the user an idea of what the user interface looks like. This creates a pretty much identical ISO image as ImgBurn, but with more information possible to dump for preservation and informational purposes. This means they could also be contributing to Redump as well as PCSX2 (like I do) 😉

I also added some various little grammar and punctuation fixes, as well as changing the inline HTML to Markdown for consistency. 😄

Lemme know if anything needs changing or updating!

### CODE REVIEW

File: content/guides/basic-setup/index.md [hunk 1]
```diff
@@ -251,8 +251,30 @@ PS2 game discs are unencrypted DVDs and CDs. So, they can be dumped quickly usin
...
-- Install ImgBurn
-- Put your PS2 game disc into a DVD drive
+- Install and run ImgBurn
+- Put your game disc into an optical drive
 - Create an image file from a disc inside ImgBurn (highlighted in screenshot below)
 
-{{< img src="./img/imgburn.webp" cols=6 >}}
+![ImgBurn](./img/imgburn.webp)
+
+## Dumping PlayStation 2 discs with Media Preservation Frontend (more advanced)
```
Comment (xTVaser) [inline content/guides/basic-setup/index.md:260]: Officially we recommend imgburn, no issue with some alternative methods being added but can we label this one as `Alternative: Dumping ....`?

Also it looks like the tool is windows only, let's add that somewhere.  This kinda just made me realize we have no image dumping instructions for linux users as well!

File: content/guides/basic-setup/index.md [hunk 2]
```diff
@@ -251,8 +251,30 @@ PS2 game discs are unencrypted DVDs and CDs. So, they can be dumped quickly usin
...
+On GitHub: <https://github.com/SabreTools/MPF>
+
+### How to use it
+
+- Extract and run MPF
+- Put your game disc into an optical drive
+- Select PS2 for system/media type
+- Select whether it is a CD-ROM PS2 game or DVD-ROM PS2 game
+- Select output file name and directory
+- Select the apporopiate drive letter for the drive with the optical media in it
```
Comment (xTVaser) [inline content/guides/basic-setup/index.md:275]: s/apporopiate/appropriate

File: content/guides/basic-setup/index.md [hunk 3]
```diff
@@ -251,8 +251,30 @@ PS2 game discs are unencrypted DVDs and CDs. So, they can be dumped quickly usin
...
+- Select "Start dumping"
+
+One quick note, the lower the drive speed of the optical drive the more likely you are to get a good dump of the disc. The fastest drive speed will take much less time, but may be incomplete, corrupted, or inaccruate. It could also be perfectly fine, so depending on your drive and your media, try different things and see what works best for you.
```
Comment (xTVaser) [inline content/guides/basic-setup/index.md:278]: s/inaccruate/inaccurate

**Review by xTVaser — CHANGES_REQUESTED** [review]
Sorry this has sat for so long.

Thanks for the general cleanup, it all looks good.  I just had one small suggestion / found a few typos before merging.  You'll have to resolve conflicts after some instructions recently got added to this guide as well.

File: content/guides/basic-setup/index.md (hunk 1 above)
Comment (DerekTurtleRoe) [reply content/guides/basic-setup/index.md:260]: Sorry for the late response, I forgot that I sent this PR and then the reply got lost in my notifications. 😩 

Yeah, I can do that for sure! Linux support for this tool is in development, so at some point we might be able to reuse that info for Linux and Mac as well.

File: content/guides/basic-setup/index.md (hunk 2 above)
Comment (DerekTurtleRoe) [reply content/guides/basic-setup/index.md:275]: Will fix!

File: content/guides/basic-setup/index.md (hunk 3 above)
Comment (DerekTurtleRoe) [reply content/guides/basic-setup/index.md:278]: Will fix this as well! 😄

**Comment by DerekTurtleRoe** [comment]
@xTVaser Alright, lemme know how this looks!

**Comment by xTVaser** [comment]
thanks!

**Pull request merged**